tests/main/views/test_marketplace.py

```diff
@@ -1775,33 +1775,42 @@ def test_live_results_returns_valid_json_structure(self):
         res = self.client.get('/digital-outcomes-and-specialists/opportunities?live-results=true')
         data = json.loads(res.get_data(as_text=True))
 
-        assert set(data.keys()) == {
+        assert sorted(data.keys()) == sorted((
```

Comment:
is this just because it gives better pytest output on failure?

Reply:
Na I was thinking it probably _shouldn't_ pass if it ends up with something like:
```
set(('foo', 'bar', 'foo')) == {'foo', 'bar'}
```
I'm guessing `set` was used because of the implicit sorting but it could result in false positives.

Reply:
Oh I get it! It's about repetitions. Cool.